bycycle/burst/amp.py

```diff
@@ -22,6 +22,19 @@ def detect_bursts_amp(df_features, burst_fraction_threshold=1, min_n_cycles=3):
     -------
     df_features : pandas.DataFrame
         Dataframe updated, with a additional column to indicate if the cycle is part of a burst.
+
+    Examples
+    --------
+    Apply thresholding for dual amplitude burst detection.
+
+    >>> from bycycle.features import compute_burst_features, compute_shape_features
```

Comment:
I made comments on organizing args in `burst/cycle` below. Not too important how you choose to organize them, but if you do update, please check through and make consistent in all examples that have the same layout (importing and using sim_bursty_oscillation, etc).